schemas/HPXMLBaseElements.xsd

```diff
@@ -4368,6 +4468,8 @@
 			<xs:element name="Appliances" type="Appliances" minOccurs="0"/>
 			<xs:element name="Lighting" type="Lighting" minOccurs="0"/>
 			<xs:element minOccurs="0" name="Pools" type="Pools"/>
+			<xs:element minOccurs="0" name="StationarySpas" type="StationarySpas"/>
+			<xs:element minOccurs="0" name="PortableSpas" type="PortableSpas"/>
```

Comment:
Instead of separate `StationarySpas/StationarySpa` and `PortableSpas/PortableSpa`, could we have `StationarySpa` and `PortableSpa` inside a single `Spas` parent?